python/ray/autoscaler/autoscaler.py

```diff
@@ -103,6 +104,9 @@
     "file_mounts": (dict, OPTIONAL),
 
     # List of common shell commands to run to initialize nodes.
+    "startup_commands": (list, OPTIONAL),
```

Comment:
@richardliaw Does this seem reasonable to you?

Reply:
The `startup_commands` are pretty much the same as `setup_commands` except that they are run on the host instead of the docker container. One way to achieve the same functionality would be to always run the `setup_commands` on the host instead of docker, i.e. require the docker commands to be handled explicitly. I don't think that would be unreasonable either.

Reply:
yeah I think this is fine; the main nit I have is that I'd find having both `startup_commands` and `setup_commands` confusing... is there some other naming that indicates that these commands will be run directly on the instance (and not wrapped in any way, unlike setup_commands)?

Reply:
I agree. That would be solved by combining startup and setup commands and making them always run on host. I'm not sure what to call these to not make them confusing. Maybe `initialization_commands` and `setup_commands`?

Reply:
yeah that works